src/main/java/com/google/devtools/build/lib/analysis/RunfilesSupport.java

```diff
@@ -83,6 +84,7 @@
   private final Artifact owningExecutable;
   private final boolean createSymlinks;
   private final CommandLine args;
+  private final Map<String, String > envs;
```

Comment:
Make this an `ImmutableMap`; otherwise, `RunfilesSupport` would not be immutable.